src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -71,8 +77,11 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
     when(fileMetadataRepositoryMock.addFileMetadata(any[Seq[FilemetadataRow]])).thenReturn(mockFileMetadataResponse)
     val fileMetadataService = new FileMetadataService(fileMetadataRepositoryMock, FixedTimeSource, fixedUuidSource)
```

Comment:
Is it worth abstracting the services creation out from the tests themselves as there is a lot of repeated code in the tests? Similar to the mock repository set up.

Reply:
I've extracted it out to a val. There's two tests that need their local copies of uuidSource passed to the service to work properly so I've created new services for those tests but every other test is using the new global val in the class.